django_app/redbox_app/templates/support.html

```diff
@@ -28,15 +24,13 @@ <h2 class="govuk-heading-m">Signing in</h2>
 
   <p class="govuk-body">You should check your junk or spam folder for an email from Redbox Copilot.</p>
 
-  <p class="govuk-body">If you're still having trouble, ask your line manager for support. If the user guide doesn't have the answer then you can contact us using the details at the bottom of this page.</p>
+  <p class="govuk-body">If you're still having trouble then you can contact us using the details at the bottom of this page.</p>
 
- <h2 class="govuk-heading-m">Email address not allowed</h2>
+  <h2 class="govuk-heading-m">Email address not allowed</h2>
```

Comment:
Could you add a ticket to the backlog to come update this once the auth solution is in place please?

Reply:
Yep, good shout.

Reply:
https://technologyprogramme.atlassian.net/jira/software/projects/REDBOX/boards/409?selectedIssue=REDBOX-159